offload/cmake/caches/AMDGPUBot.cmake

```diff
@@ -0,0 +1,14 @@
+set(LLVM_ENABLE_PROJECTS "clang;lld" CACHE STRING "")
+set(LLVM_ENABLE_RUNTIMES "compiler-rt;openmp;offload" CACHE STRING "")
+set(LLVM_ENABLE_PER_TARGET_RUNTIME_DIR ON CACHE BOOL "")
+set(LLVM_ENABLE_ASSERTIONS ON CACHE BOOL "")
+set(LLVM_LIT_ARGS "-vv --show-unsupported --timeout 100 --show-xfail -j 64" CACHE STRING "")
+# set(LLVM_RUNTIME_TARGETS default;amdgcn-amd-amdhsa CACHE STRING "")
+set(LLVM_TARGETS_TO_BUILD X86;AMDGPU CACHE STRING "")
```

Comment:
Looks weird with no quotes around `X86;AMDGPU`. Does CMake not require them?

Reply:
AFAIK quotes are only necessary on the command line because bash interprets `;` as an end of line.